### PR TITLE
Linked accounts update

### PR DESCRIPTION
**Pull request by JonSzeto821**

### CODE REVIEW

File: themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx [hunk 1]
```diff
@@ -132,15 +126,15 @@ class LinkedAccountsPage extends React.Component<LinkedAccountsPageProps, Linked
                 <PageSection isFilled variant={PageSectionVariants.light}>
                     <Stack hasGutter>
                         <StackItem>
-                            <Title headingLevel="h2" size='2xl'>
+                            <Title headingLevel="h2" className="pf-u-mb-lg" size='xl'>
```
Comment (srambach) [inline themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx:129]: If you wrap a <TextContent> around the <Title> then it should give you the correct vertical spacing without manually adding the utility class.

File: themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx [hunk 2]
```diff
@@ -132,15 +126,15 @@ class LinkedAccountsPage extends React.Component<LinkedAccountsPageProps, Linked
...
                                 <Msg msgKey='linkedLoginProviders'/>
                             </Title>
                             <DataList id="linked-idps" aria-label='foo'>
                                 {this.makeRows(this.state.linkedAccounts, true)}
                             </DataList>
                         </StackItem>
                         <StackItem>
-                            <Title headingLevel="h2" size='2xl'>
+                            <Title headingLevel="h2" className="pf-u-mt-xl pf-u-mb-lg" size='xl'>
```
Comment (srambach) [inline themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx:137]: Try adding a <TextContent> around the <Title> and then remove the utility classes.

File: themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx [hunk 3]
```diff
@@ -185,9 +179,22 @@ class LinkedAccountsPage extends React.Component<LinkedAccountsPageProps, Linked
...
+                                            <SplitItem className="pf-u-my-xs" isFilled><span id={`${account.providerAlias}-idp-label`}>{this.label(account)}</span></SplitItem>
+                                        </Split>
+                                    </DataListCell>,
+                                    <DataListCell key='username' width={5}>
+                                        <Split>
+                                            <SplitItem className="pf-u-my-xs" isFilled><span id={`${account.providerAlias}-idp-username`}>{account.linkedUsername}</span></SplitItem>
+                                        </Split>
+                                    </DataListCell>,
                                 ]}/>
                             <DataListAction aria-labelledby='foo' aria-label='foo action' id='setPasswordAction'>
```
Comment (srambach) [inline themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx:199]: Do you still want "foo" here?

File: themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx [hunk 4]
```diff
@@ -185,9 +179,22 @@ class LinkedAccountsPage extends React.Component<LinkedAccountsPageProps, Linked
...
+                                    <DataListCell key='label'>
+                                        <Split>
+                                            <SplitItem className="pf-u-my-xs" isFilled><span id={`${account.providerAlias}-idp-label`}>{this.label(account)}</span></SplitItem>
+                                        </Split>
+                                    </DataListCell>,
+                                    <DataListCell key='username' width={5}>
+                                        <Split>
+                                            <SplitItem className="pf-u-my-xs" isFilled><span id={`${account.providerAlias}-idp-username`}>{account.linkedUsername}</span></SplitItem>
+                                        </Split>
+                                    </DataListCell>,
```
Comment (srambach) [inline themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx:197]: You probably don't need these vertical margin utilities - are you reducing the margin? If so you could try the compact variant of the data list. (I'm not familiar with what you are implementing here though)

**Review by srambach — COMMENTED** [review]
Great working through some layouts with you! I am not totally familiar with the screen you are doing here but I'm happy to talk more if my comments don't address the issues.

File: themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx (hunk 4 above)
Comment (JonSzeto821) [reply themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx:197]: this was mainly used to position a text element to center it vertically next to a logo image. I tried looking at the isCompact prop, but that looks to only get applied on the DataList component. Having that applied compacts all of the white space within each row - which we need in this scenario

File: themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx (hunk 3 above)
Comment (JonSzeto821) [reply themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx:199]: I've updated all references of foo to have a more context-fitting name

File: themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx (hunk 1 above)
Comment (JonSzeto821) [reply themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx:129]: I am not able to achieve the correct spacing with the <TextContent><Text> components as suggested. It doesn't appear to apply the correct spacing the spacing utility class provides. May need to meet and look over together.

**Review by tyandor — APPROVED** [review]

**Pull request merged**